src/store/modules/requests/index.js

```diff
@@ -0,0 +1,15 @@
+import actions from './actions.js';
+import mutations from './mutations.js';
+import getters from './getters.js';
+
+export default {
+  namespaced: true,
+  state() {
+    return {
+      requests: [],
+    }
+  },
+  actions,
+  mutations,
+  getters,
+}
```

Comment:
@HVRolim newline no final do arquivo
```suggestion
}

```